exercicios/para-casa/entregas/Lucelia-Batista/resolucao.test.js

```diff
@@ -0,0 +1,31 @@
+const contaBancaria = require("./resolucao")
+
+describe("Conta Bancária", () => {
+    let conta
+    beforeEach(() => {
+        conta = new contaBancaria("Luiz", 10000, 2)
+    })
+
+    // Consulta
+    test("Validar consulta do saldo", () => { expect(conta.consultarSaldoAtual()).toEqual(10000) })
+
+    // Saque
+    test("Validar erro de saldo insuficiente", () => {
+        expect(() => { conta.retirarValor(11000) }).toThrow()
```

Comment:
```suggestion
        expect(() => { conta.retirarValor(11000) }).toThrow("Saldo insuficiente");
```
Ao testar se uma exceção é lançada, é útil adicionar uma mensagem de erro descritiva para explicar o que está sendo testado. Isso facilita a depuração, caso um teste falhe.